docs/en-US/commands/Find-JiraFilter.md

```diff
@@ -0,0 +1,298 @@
+---
+external help file: JiraPS-help.xml
+Module Name: JiraPS
+online version: https://atlassianps.org/docs/JiraPS/commands/Find-JiraFilter/
+locale: en-US
+schema: 2.0.0
+layout: documentation
+permalink: /docs/JiraPS/commands/Find-JiraFilter/
+---
+# Find-JiraFilter
+
+## SYNOPSIS
+
+Find JIRA filter(s).
+
+## SYNTAX
+
+### ByInputObject
+
+```powershell
+Find-JiraFilter [[-Name] <string[]>] [[-AccountId] <string>] [[-GroupName] <string>] [[-Project] <Object>] [[-Fields] {description | favourite | favouritedCount | jql | owner |
+    searchUrl | sharePermissions | subscriptions | viewUrl}] [[-Sort] <string>] [[-Credential] <pscredential>] [-IncludeTotalCount] [-Skip <uint64>] [-First <uint64>]
+    [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Searches for filters. This operation is similar to Get filters except that the results can be refined to include filters that have specific attributes. For example, filters with a particular name. When multiple attributes are specified only filters matching all attributes are returned.
```

Comment:
Add a disclaimer that this only works with API version 3, which is not available for non-cloud Jira Server (at the time of this writing)

Reply:
I added the following disclaimer, since Jira Cloud supports it in both v2 and v3.

> This works with Jira Cloud only.  It does not work with non-cloud Jira Server (v8.3.1 at the time of this writing).